style.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+
+.container {
+  position: relative;
+  width: fit-content;
+}
```

Comment:
Please note that your page still has a scroll. I suggest you get rid of this just as the previous reviewer highlighted. 
Remove the `position` and also git it a widht of 100%
Also note that the position of the humburger icon is no longer on the right as it should be.

- [ ] This is your page
<img width="364" alt="Screenshot 2022-09-24 at 17 09 30" src="https://user-images.githubusercontent.com/48546992/192102620-66655a17-4dc9-48a5-a09a-ea66c4943bd2.png">

- [ ] This is the figma design
<img width="376" alt="Screenshot 2022-09-24 at 17 09 55" src="https://user-images.githubusercontent.com/48546992/192102632-db2e9ceb-9677-4986-8be8-eaf2cd294ba8.png">